src/main/groovy/org/shipkit/internal/notes/header/BufferedWriterWrapper.java

```diff
@@ -0,0 +1,51 @@
+package org.shipkit.internal.notes.header;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class BufferedWriterWrapper {
+
+    private final BufferedWriter bufferedWriter;
+
+    public BufferedWriterWrapper(File file) {
+        try {
+            bufferedWriter = new BufferedWriter(new FileWriter(file));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create FileWritter for " + file, e);
+        }
+    }
+
+    public void write(String str) {
+        try {
+            bufferedWriter.write(str);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file", e);
```

Comment:
If you are able to defend the need to use the wrapper types, it would be great if those exception messages also include the file path.

Ignore my feedback if we are getting rid of the wrapper types ;)